crane/app_util.py

```diff
@@ -71,9 +71,16 @@ def repo_is_authorized(repo_id):
 
     if repo_tuple.protected:
         cert = _get_certificate()
-        if not cert or not cert.check_path(repo_tuple.url_path):
+        logger.info('cert when checking repo %s' % cert)
+        value = repo_tuple.url_path
+        logger.info('path before check %s' % value)
+        if 'webassets/docker' in value:
```

Comment:
This string is repeated in several places. Should it be stored in a variable at module scope, or perhaps in configuration?

Alternatively, perhaps this logic should factored into a function that can be called from the various places it is used, like: `if not cert or not cert.check_path(remove_prefix(path)): ...`.

Reply:
Since we are planning to move to Openshift registry, I think this patch is only required for this release and eventually, it would be reverted/removed when we move to Openshift registry.
So, I thought this was an extra effort to make it more modularize